### ISSUE TITLE
MaxSdkCallbacks not trigger on newest version

### ISSUE THREAD

**Issue by tracer8** [AppLovin/AppLovin-MAX-Unity-Plugin#358]
### MAX Plugin Version

6.4.2

### Unity Version

2020.3.42f1

### Device/Platform Info

All

### Current Behavior

MaxSdkCallbacks is not trigger on Unity MAX SDK v6.4.2.
All events of inter, reward... is not trigger after i upgrade to v6.4.2

### How to Reproduce

- Upgrade to MAX SDK v6.4.2
- Add a call back like: MaxSdkCallbacks.Rewarded.OnAdLoadedEvent += OnRewardedAdLoadedEvent;
- Try to show in editor and check the debug log

**Comment by santoshbagadi** [comment]
Closing this as duplicate of: https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/issues/353#issuecomment-2041157501

**Comment by tracer8** [comment]
I want to ask something. Can i just use older version  with newest adapter
update?
And wait for sdk better to upgrade?

**Comment by santoshbagadi** [comment]
Yes, our adapters are backwards compatible and you should be able to update the adapters without updating our SDK.

Apologies for the inconvenience and we'll have a release with the fix soon.
